src/app/view/mainMenu.js

```diff
@@ -0,0 +1,39 @@
+const nav = document.querySelector('.header__game-nav');
+const tab = document.getElementsByClassName('button');
+
+const students = document.createElement('button');
```

Comment:
You can use our function elementCreator from src/app/logic/elementCreator.js

Reply:
It's in utils. I look at it, but for now I don't understand it... Could you explain it?